src/game_interpreter.cpp

```diff
@@ -821,6 +821,8 @@ bool Game_Interpreter::ExecuteCommand(lcf::rpg::EventCommand const& com) {
 			return CommandManiacSetGameOption(com);
 		case Cmd::Maniac_CallCommand:
 			return CommandManiacCallCommand(com);
+		case static_cast<Game_Interpreter::Cmd>(2053): //Cmd::EasyRpg_SetFlag
+			return CommandSetFlag(com);
```

Comment:
Can you rename this to ``CommandEasyRpgSetInterpreterFlag``?

Reply:
renamed.